src/domUpdates.js

```diff
@@ -31,17 +33,17 @@ function hide(elements) {
 }
 
 function showLoginPage() {
-show([loginSection]);
-hide([calendarView, roomFilterDropDown, allBookingsTitle, bookingSection, availableTitle, availableToBookSection, userMessage, clickForRooms]);
+show([loginSection, loginHere]);
+hide([calendarView, roomFilterDropDown, allBookingsTitle, bookingSection, availableTitle, availableToBookSection, userMessage, clickForRooms, hero]);
```

Comment:
This is a lot of stuff. Maybe this is a bad suggestion, but what if you wrapped it all in a div and then just hid the whole dang thing?